src/app/api/analytics/stats.py

```diff
@@ -290,65 +288,10 @@ def has_license_wrong_entries(entry: str, properties: dict) -> bool:
     return False
 
 
-def materials_with_missing_properties(
+def material_validation(
```

Comment:
rename node_id argument to collection_id to indicate it is not a material_id?

Reply:
Renamed